dotCMS/src/main/java/com/dotcms/filters/interceptor/cas/CasAutoLoginWebInterceptor.java

```diff
@@ -1,7 +1,10 @@
 package com.dotcms.filters.interceptor.cas;
 
+import com.dotcms.business.CloseDBIfOpened;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.business.CloseDBIfOpened'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)